proto/conduit/v1/schema.proto

```diff
@@ -15,18 +15,15 @@ message Schema {
     TYPE_AVRO = 1;
   }
```

Comment:
Can we move this `Schema` message to conduit-commons? We need it in processors as well.

Reply:
@lovromazgon Added [here](https://github.com/ConduitIO/conduit-commons/pull/67/commits/20efa2b33ee80c327818950bafae9f6915db298b)